### PR TITLE
Feat/Job offer page

### PR DESCRIPTION
**Pull request by jphinning**
- Job offers for the freelancer page done as can be seen below 👇🏻  Images are still placeholders, and will be integrated in a later PR.
image

- Offer details
image

### CODE REVIEW

File: src/components/JobOffers/JobOffersStyles.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+  DARK_BLUE,
+  TEXT_GRAY,
+  WHITE,
+  TAG_CLOSED,
+} from "utils/consts/colorConsts";
+
+export const Wrapper = styled.div`
+  display: flex;
+  border-top: 1px solid ${BOX_BORDER};
+  /* align-items: center; */
```
Comment (xx-TOOL-xx) [inline src/components/JobOffers/JobOffersStyles.tsx]: uncomment or remove useless styles, same to another commented styles in this file

File: src/components/UI/drawers/JobOfferDrawer/JobOfferDrawer.tsx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+          <Header>
+            <Title level={3}>Job Offer</Title>
+            <CustomText>{dayjs(jobPost.createdAt).fromNow()}</CustomText>
+          </Header>
+
+          <ContentBox>
+            <Title level={5}>{t("Offer.drawerTitle")}</Title>
+            <Space direction="vertical">
+              <CustomText strong color={BLUE}>
+                {status.charAt(0).toUpperCase() + status.slice(1)}
```
Comment (xx-TOOL-xx) [inline src/components/UI/drawers/JobOfferDrawer/JobOfferDrawer.tsx]: As I understand u trying to make first letter capital, but it not the js case, I believe. Can it be done by CSS `text-transform: capitalize;`?

File: src/components/JobOffers/JobOffers.tsx [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+                  {t("Offer.acceptOffer")}
+                </AcceptButton>
+                <RejectButton onClick={handleClose}>
+                  {t("Offer.rejectOffer")}
+                </RejectButton>
+              </>
+            )}
+            <TagStatus>
+              <StatusTag tag={OfferTags[status]}>
+                {status.charAt(0).toUpperCase() + status.slice(1)}
```
Comment (xx-TOOL-xx) [inline src/components/JobOffers/JobOffers.tsx]: I lefted comment for same functionality in this PR in JobOfferDrawer.tsx

File: src/components/JobOffers/JobOffers.tsx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+          <CustomText strong color={BLUE}>
+            {jobPost.category?.name}
+          </CustomText>
+          {" - "}
+          {jobPost.duration} {jobPost.durationType} duration
+          {" - "}
+          {dayjs(jobPost.createdAt).fromNow()}
+        </DeatailedInfo>
+        <JobDescription>
+          {"Offer: "}
```
Comment (xx-TOOL-xx) [inline src/components/JobOffers/JobOffers.tsx]: Localization?

**Review by xx-TOOL-xx — APPROVED** [review]

File: src/components/JobOffers/JobOffersStyles.tsx (hunk 1 above)
Comment (jphinning) [reply src/components/JobOffers/JobOffersStyles.tsx]: Done

File: src/components/UI/drawers/JobOfferDrawer/JobOfferDrawer.tsx (hunk 2 above)
Comment (jphinning) [reply src/components/UI/drawers/JobOfferDrawer/JobOfferDrawer.tsx]: Good idea

**Comment by lyudad** [comment]
Снимок экрана 2022-10-04 в 14 23 07
I dont like it. its not obvious what is status what are buttons

**Comment by lyudad** [comment]
2) offer cant be active, if offer accepted it become contract, it could be accepted but not active

**Review by lyudad — APPROVED** [review]

**Comment by jphinning** [comment]
- Accepted offers won't be shown in the offers list, only on the contracts page
- Hook to fetch information about both job offer components created 
- Ux updated, buttons will now appear below the offer value
- Expired tag added
- Detailed drawer updated with expiring information
image
image

**Review by lyudad — APPROVED** [review]

**Pull request merged**